Analysis-file/LngLat.sql

```diff
@@ -0,0 +1,51 @@
+BEGIN TRANSACTION;
+
+-------------------
+-- Turn timing on so we can see how long it took
+-------------------
+\timing
+
+DROP TABLE IF EXISTS cqcref.postcodes;
+
+SELECT 'Creating new postcodes table';
+
+CREATE TABLE cqcref.postcodes (
+    postcode character varying(10) COLLATE pg_catalog."C",
+    longitude double precision,
+    latitude double precision
+);
+
+SELECT 'Copy from CSV to the table';
+
+\COPY cqcref.postcodes (postcode, longitude, latitude) FROM '/mnt/c/Users/arussell/Documents/Repos/SFC-DB/Analysis-file/postcodes.csv' WITH (FORMAT csv, ENCODING 'UTF8');
+
+SELECT 'Update the workers lat and long depending on their postcode';
+
+UPDATE cqc."Worker"  wrk
+SET 
+    (
+        "Longitude",
+        "Latitude"
+    ) = (
+        lnglat.longitude,
+        lnglat.latitude
+    )
+FROM cqcref.postcodes lnglat
```

Comment:
Do you think the alias should be short for `postcodes` rather than `lnglat`?

`FROM cqcref.postcodes pc` or something?

Reply:
Not sure really, we want long and lat from it, that's why i called it that.

Reply:
It's usually to save typing out the whole table name when using it as a reference but I don't think there's a specific convention to use.

Reply:
So are you happy to leave as is?